example/simple/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.12.4) #TODO replace with generated version
+cmake_minimum_required(VERSION 3.12.4)
```

Comment:
I decided that currently the overhead for generating the CMakeLists is too high, compared to manually updating them.